### PR TITLE
Further update models

### PR DESCRIPTION
**Pull request by sarakonradi**
This PR is based on #1374 and further updates the model files.

It: 
- Ensures keywords are consistently written in singular form
- Writes out abbreviations (e.g. `AC` and `DC`)
- Updates the `Device name`
- Adds an empty line between `EndUserDocs` and the class definition
- Corrects model names, such as `iaf_psc_exp_htum`

### CODE REVIEW

File: models/ac_generator.h [hunk 1]
```diff
@@ -46,7 +46,7 @@ ac_generator
 Description
 +++++++++++
 
-This device produces an ac-current sent by a CurrentEvent. The
+This device produces an AC current sent by a CurrentEvent. The
```
Comment (jougs) [inline models/ac_generator.h]:
```suggestion
This device produces AC input sent by CurrentEvents. The
```

Reasoning: AC already has a 'current' in it, cf. [PIN number, LCD display, ...](https://en.wikipedia.org/wiki/RAS_syndrome)

File: models/bernoulli_connection.h [hunk 2]
```diff
@@ -58,8 +58,8 @@ Parameters
 
 FirstVersion: June 2017
 
-Transmits
-+++++++++
+Sends
```
Comment (jougs) [inline models/bernoulli_connection.h]: This needs to stay `Transmits`. Also in other files ending in `_connection.h`.

The logic is that neurons have the (single) event type they will send during the simulation under `Sends` and all event types they are able to handle under `Receives`. As synapses/connections only relay the events from the sender to the receiver, the have everything they can relay under `Transmits`.

File: models/dc_generator.h [hunk 3]
```diff
@@ -38,15 +38,20 @@
...
+Device name
++++++++++++
+
+dc_generator
 
 Description
 +++++++++++
 
-The DC-Generator provides a constant DC Input to the connected node. The unit of the current is pA.
+The dc_generator provides a constant DC Input to the connected node. The unit of the current is pA.
```
Comment (jougs) [inline models/dc_generator.h]:
```suggestion
The dc_generator provides a constant DC input to the connected node. The unit of the current is pA.
```

**Review by jougs — CHANGES_REQUESTED** [review]
Many thanks! This looks mostly good to me :-)

For the future, could you please @mention me (by writing `@jougs` in the description of the PR or a comment somewhere) when you're doing these kinds of things? That way I'd be alerted by email when something is ready for me to look at without any need to communicate that via other means. Thanks!

**Comment by sarakonradi** [comment]
@jougs Thanks for your suggestions! 

I added the `AC input`, `DC input` and `Transmits` changes. 

RAS syndrome can be painful. Hence, I removed the repetitions.

**Comment by jougs** [comment]
Great!

Can you please again pull `jessica-mitchell:update_models_rst_format` into your branch and fix the conflicts? From my side this would then be good to go.

**Comment by jougs** [comment]
@jessica-mitchell: could you please merge this? I'm fine with the changes now.
@sarakonradi: Merging this one into the branch nest/nest-simulator#1374 is based off should make Travis happy.

**Pull request merged**